src/solver.js

```diff
@@ -1154,8 +1163,13 @@ Tableau.prototype.setEvaluation = function () {
     // Rounding objective value
     var roundingCoeff = Math.round(1 / this.precision);
     var evaluation = this.matrix[this.costRowIndex][this.rhsColumn];
-    this.evaluation =
+    var roundedEvaluation =
         Math.round(evaluation * roundingCoeff) / roundingCoeff;
+
+    this.evaluation = roundedEvaluation;
+    if (this.simplexIters === 0) {
+        this.relaxedSolution = roundedEvaluation;
```

Comment:
Alright, that makes sense. However, the value of the variable `relaxedEvaluation` needs to be equal to the value found in the very first run of the simplex algorithm, as we want to compare the MILP evaluation to this variable later. You might have a better idea where and how to implement this part though, as I'm still relatively new to JS development.